ext/console/02_console.js

```diff
@@ -326,7 +326,7 @@
   }
 
   function maybeColor(fn, inspectOptions) {
-    return inspectOptions.colors ? fn : (s) => s;
+    return inspectOptions.colors && !colors.getNoColor() ? fn : (s) => s;
```

Comment:
I believe this might have been on purpose, to always be able to force using colors, though I might be wrong...